frontend/index.html

```diff
@@ -117,5 +100,27 @@
         })
         .forEach((link) => document.head.append(link));
     </script>
+
+    <script
+      type="text/javascript"
+      src="https://preview.vaadin.com/vaadincom/haas-service/haas-loader.js"
+      defer={true}
+      key="haas-loader"
+    ></script>
+
+    <!-- Here are the style sheets that HaaS loads (check latest from HubSpot):
```

Comment:
These can be removed, they are uninteresting and will be out of date soon anyway

Reply:
Removed